### DISCUSSION TITLE
Good toonsquid beginner friendly keyframe tutorial?

### DISCUSSION THREAD

**Discussion by MustardMonday** [keiwando/toonsquid#430] [Q&A]
I'm trying to expand form my basic frame by frame starter animation into keyframing but am really not the best at figuring out how it works, even with watching many videos on it. Are there any incredibly beginner friendly toonsquid tutorials on keyframes which would be recommend?

**Comment by rowby** [comment]
For such a really outstanding app I was surprised to see not a whole lot of ToonSquid video tutorials.

Fortunately I found a series excellent of YouTube videos by Stayf Draws.  

I cannot get the GetHub url link feature to work.  But just search YouTube for Stayf Draws.  Then go to his playlists — and you’ll see about 12 lessons focused on ToonSquid.

He focuses on both transitional frame-by-frame  and keyframe animation using ToonSquid.

…Rowby.

**Reply by ota-works** [reply]
Yeah, it's really sad, tbh.

I wish people would make more high-quality tutorials for Toonsquid too
